app/models/release.rb

```diff
@@ -35,9 +39,28 @@ def author
   end
 
   def self.find_by_version(version)
-   if version =~ /\Av(\d+)\Z/
-     number = $1.to_i
-     find_by_number(number)
-   end
- end
+    if version =~ /\Av(\d+)\Z/
+      number = $1.to_i
+      find_by_number(number)
+    end
+  end
+
+  private
+
+  def assing_release_number
+    latest_release_number = project.releases.last.try(:number) || 0
```

Comment:
they should be always in sync, but number might not be the same as count when you delete 1 or somehow else start at a higher number